modal/modal.html

```diff
@@ -9,142 +9,142 @@
                 data-share="https://pinboard.in/add?showtags=yes&amp;url&amp;title">
             <img src="icons/pinboard.png" />
         </button>
 
-        <button class="col share clearfix" id="mewe" data-share="http://mewe.com/share?link">
+        <button class="col share clearfix" id="mewe" title=“MeWe” data-share="http://mewe.com/share?link">
             <img src="icons/mewe.png" />
         </button>
 
-        <button class="col share clearfix" id="outlook"
+        <button class="col share clearfix" id="outlook" title=“Outlook”
                 data-share="https://outlook.live.com/owa/?path=/mail/action/compose&body&subject">
             <img src="icons/outlook.svg" />
         </button>
 
-        <button class="col share clearfix" id="weibo"
+        <button class="col share clearfix" id="weibo" title=“Weibo”
                 data-share="https://service.weibo.com/share/share.php?url&title">
             <img src="icons/weibo.svg" />
         </button>
 
-        <button class="col share clearfix" id="skype"
+        <button class="col share clearfix" id="skype" title=“Skype”
                 data-share="https://web.skype.com/share?url">
             <img src="icons/skype.svg" />
         </button>
 
-        <button class="col share clearfix" id="whatsapp"
+        <button class="col share clearfix" id="whatsapp" title=“WhatsApp”
                 data-share="https://wa.me/?text">
             <img src="icons/whatsapp.svg" />
         </button>
 
-        <button class="col share clearfix" id="wayback" data-share="">
+        <button class="col share clearfix" id="wayback" title=“Wayback” data-share="">
             <img src="icons/wayback.svg" />
         </button>
 
-        <button class="col share clearfix" id="pinterest"
+        <button class="col share clearfix" id="pinterest" title=“Pinterest”
                 data-share="https://www.pinterest.it/pin/create/button/?url">
             <img src="icons/pinterest.svg" />
         </button>
 
-        <button class="col share clearfix" id="evernote"
+        <button class="col share clearfix" id="evernote" title=“Evernote”
                 data-share="https://www.evernote.com/clip.action?url">
             <img src="icons/evernote.svg" />
         </button>
 
-        <button class="col share clearfix" id="instapaper"
+        <button class="col share clearfix" id="instapaper" title=“Instapaper”
                 data-share="https://www.instapaper.com/edit?url">
             <img src="icons/instapaper.png" />
         </button>
 
-        <button class="col share clearfix" id="scoopit"
+        <button class="col share clearfix" id="scoopit" title=“Scoop.it”
                 data-share="https://www.scoop.it/bookmarklet?url">
             <img src="icons/scoopit.png" />
         </button>
 
-        <button class="col share clearfix" id="bingtranslate"
+        <button class="col share clearfix" id="bingtranslator” title=“Bing Translator” 
```

Comment:
If you change the id, you need to to the same in the options.html or the settings won't match.